src/modules/facility-investor/facility-investor.controller.ts

```diff
@@ -62,4 +66,33 @@ export class FacilityInvestorController {
     await this.facilityInvestorService.createInvestorForFacility(facilityIdentifier, facilityInvestorToCreate);
     return Promise.resolve(new CreateFacilityInvestorResponse(facilityIdentifier));
   }
+
+  @Get('facilities/:facilityIdentifier/investors')
+  @ApiOperation({
+    summary: 'Get all investors of the facility.',
+  })
+  @ApiParam({
+    name: 'facilityIdentifier',
+    required: true,
+    type: 'string',
+    description: 'The identifier of the facility in ACBS.',
+    example: EXAMPLES.FACILITY_ID,
+  })
+  @ApiOkResponse({
+    description: 'The facility investors have been successfully retrieved.',
+    type: GetFacilityInvestorResponseItem,
+    isArray: true,
+  })
+  @ApiNotFoundResponse({
+    description: 'The facility was not found.',
+  })
+  @ApiBadRequestResponse({
+    description: 'Bad request.',
+  })
+  @ApiInternalServerErrorResponse({
+    description: 'An internal server error has occurred.',
+  })
+  async getInvestorsForFacility(@Param() params: GetFacilityInvestorsParamsDto): Promise<GetFacilityInvestorsResponse> {
+    return await this.facilityInvestorService.getInvestorsForFacility(params.facilityIdentifier);
```

Comment:
veeery small item: we don't need the `return await`, we can ` return this.facilityInvestorService...`  directly